src/main/kotlin/no/nav/familie/baks/mottak/søknad/NyPdfService.kt

```diff
@@ -0,0 +1,124 @@
+package no.nav.familie.baks.mottak.søknad
+
+import no.nav.familie.baks.dokgen.DokGen
+import no.nav.familie.baks.mottak.integrasjoner.FamilieDokumentClient
+import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.DBBarnetrygdSøknad
+import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.SøknadV7
+import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.SøknadV8
+import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.VersjonertSøknad
+import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.DBKontantstøtteSøknad
+import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.KontantstøtteSøknadV3
+import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.KontantstøtteSøknadV4
+import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.VersjonertKontantstøtteSøknad
+import no.nav.familie.kontrakter.ba.søknad.v4.Søknadstype
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+import java.util.Locale
+
+class NyPdfService(
```

Comment:
Ny er et dårlig navn. For etterhvert så slutter den å være Ny

Reply:
Fikset det med forslaget ditt under